kv/task.go

```diff
@@ -951,6 +951,21 @@ func (s *Service) findRuns(ctx context.Context, tx Tx, filter influxdb.RunFilter
 	if filter.Limit < 0 || filter.Limit > influxdb.TaskMaxPageSize {
 		return nil, 0, influxdb.ErrOutOfBoundsLimit
 	}
+	parsedFilterAfterTime := time.Time{}
+	parsedFilterBeforeTime := time.Now().UTC()
+	var err error
+	if filter.AfterTime != "" {
```

Comment:
nit: the following is a marginally faster check for the same thing:
```suggestion
	if len(filter.AfterTime) > 0 {
```